src/main/resources/roleconfig/role_sscs.json

```diff
@@ -295,10 +333,10 @@
     ]
   },
   {
```

Comment:
"superuser" is removed, is this correct?

Reply:
yeah. `superuser` was in the old SSCS config. but its been removed from the SSCS config page by the BA